harmony-fe/src/models/album.tsx

```diff
@@ -0,0 +1,8 @@
+export type Album = {
+  albumId: number
+  albumName: string
+  albumDescription?: string
+  releaseGroupType?: string
+  artistCreditId: number
+  artistCredit: number
```

Comment:
Unless you have a reason for them right now, we probably dont need these id fields for the relations